inc/saitypes.h

```diff
@@ -691,6 +691,41 @@ typedef struct _sai_segment_list_t
     sai_ip6_t *list;
 } sai_segment_list_t;
 
+/**
+ * @brief Defines a lane with its eye values with the up and down values
+ * being in mV and left and right being in mUI.
```

Comment:
I am not sure about mUI. https://en.wikipedia.org/wiki/Unit_interval_(data_transmission)

I think it should be UI.

Reply:
UI with int32_t will cause rounding up/down. mUI is correct.

Reply:
what does 'm' stands for here? milli UI, like UI/1000?

Reply:
Yes. m in measurements stands for milli.